docs/source/traitsui_user_manual/factories_basic.rst

```diff
@@ -923,11 +923,13 @@ reordering items within the list.
 The simple style displays a single item at a time, with small arrows on the
 right side to scroll the display. The custom style shows multiple items. The
 number of items displayed is controlled by the *rows* parameter; if the number
```

Comment:
I think the `rows` parameter is used only in WX, not in QT. Should this be mentioned here?

Reply:
Maybe a better solution for controlling the display of scrollbar is simply to respect the `rows` parameter in `QT`.

Reply:
Rows is meant to be a vertical size hint.  I can see the temptation to try and use it, but we don't support changing it dynamically at the moment, which is what would be needed for removing scrollability in the general case.  Something like the current PR is a better approach.

It wouldn't be a bad idea to be able to support the rows parameter in the Qt backend, but that should probably be a different PR (and obviously would be ignored if Scrollable is False.